tenants/ai-example/single-model-serving/base/main/create-bucket-job.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app.kubernetes.io/component: minio
+    app.kubernetes.io/instance: minio
+    app.kubernetes.io/name: minio
+    app.kubernetes.io/part-of: minio
+    component: minio
+  name: create-minio-buckets
+  namespace: ai-example-single-model-serving
+spec:
+  selector: {}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: minio
+        app.kubernetes.io/instance: minio
+        app.kubernetes.io/name: minio
+        app.kubernetes.io/part-of: minio
+        component: minio
+    spec:
+      containers:
+      - name: create-buckets
+        image: image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-generic-data-science-notebook:2023.1
+        imagePullPolicy: IfNotPresent
+        command:
+          - /bin/bash
+        env:
+          - name: BUCKETNAME
+            value: "single-model-bucket"
+        envFrom:
+          - secretRef:
+              name: single-model-serving-ds-connection
+        args:
+          - -ec
+          - |-
+            oc get secret single-model-serving-ds-connection
```

Comment:
Instead of using the oc command here just mount the secret as an env variable to the job

Reply:
removed